packages/core/cli/src/utils/getIntegrationsFromPackage.ts

```diff
@@ -0,0 +1,23 @@
+import path from 'path';
+import fs from 'fs';
+
+const notAnIntegration = [
```

Comment:
I think it would be safer to have inclusive list, not exclusive - for external maintainers its not necessarily obvious that adding any other VSF package will make it an integration

Reply:
Ok, now it is just a:
```ts
export default (): string[] => [
  'about-you',
  'commercetools'
];
```